pravega/src/test/java/docs/javadsl/PravegaSettingsTestCase.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.javadsl;
+
+import akka.actor.ActorSystem;
+import akka.stream.alpakka.pravega.ReaderSettings;
+import akka.stream.alpakka.pravega.ReaderSettingsBuilder;
+import akka.stream.alpakka.pravega.WriterSettings;
+import akka.stream.alpakka.pravega.WriterSettingsBuilder;
+import akka.testkit.javadsl.TestKit;
+
+import static io.pravega.client.ClientConfig.ClientConfigBuilder;
+
+import io.pravega.client.stream.impl.DefaultCredentials;
+import io.pravega.client.stream.impl.UTF8StringSerializer;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.function.Function;
+
+public class PravegaSettingsTestCase {
+
+  protected static ActorSystem system;
+
+  @BeforeClass
+  public static void setup() {
+    system = ActorSystem.create();
+  }
+
+  @Test
+  public void readerSettings() {
+
+    // #reader-settings
+    ReaderSettings<String> readerSettings =
+        ReaderSettingsBuilder.create(system)
+            .clientConfigBuilder(
+                builder -> builder.enableTlsToController(true)) // ClientConfig customization
+            .readerConfigBuilder(
+                builder -> builder.disableTimeWindows(true)) // ReaderConfig customization
+            .withTimeout(3000)
```

Comment:
This can be updated with a Java and Scala DSL overrides that accept `Durations`.

i.e.

```
// JavaDSL
def withTimeout(duration: java.time.Duration)
// ScalaDSL
def withTimeout(duration: scala.concurrent.duration.Duration)	

// ScalaDSL usage
import scala.concurrent.duration._
...
.withTimeout(3.seconds)
```